config/routes.rb

```diff
@@ -3,9 +3,6 @@
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root 'homes#index'
 
-  get '/', to: "bands#index"
-  get '/bands/:id', to: "bands#index"
-
```

Comment:
For this `routes.rb` file, you can keep those paths listed in the file because it will help direct URLs to the React app. When a user interacts with the app and types into the URL, the HTTP request is first sent to the Rails routes. The `routes.rb` tries to find a route that matches the request to send it to the correct controller and action. However, not all of our pages are displayed with Rails, like `/bands/:id` is displayed in React. If the user enters that URL, the request for the webpage still goes to the Rails routes first, even though that webpage is handled by React! Our job is then to set up the Rails side of the app to receive those routes meant for React, and direct them `to` a Rails controller and action which points to a `div` with an `id` of `app`, because that’s where our React app loads in. For your routes
```
"/"
"/bands/:id"
```
We want to direct those paths `to` the `homes#index` controller#action because this action renders that React `div` we need!